### ISSUE TITLE
The endless loop should be terminated

### ISSUE THREAD

**Issue by grey27** [anson-vandoren/esp8266-captive-portal#5]
The ```captive_portal()``` func has a loop, there is no end when the configuration is complete,I think we should break after DNS is turned off,Otherwise, methods after ```portal.start()``` cannot be run.
```
if self.check_valid_wifi():
    print("Connected to WiFi!")
    self.http_server.set_ip(self.local_ip, self.ssid)
    self.dns_server.stop(self.poller)
    break
```
This is a great project, please forgive my grammar, because English is not my native language

**Comment by anson-vandoren** [comment]
Hi @grey27, thanks for submitting this, and you're absolutely correct. If you'd like to submit a PR with this change I'd be happy to merge it.

**Comment by anson-vandoren** [comment]
Fixed in #7 

Thanks, @grey27
